lib/rides/pages/search_suggestion_page.dart

```diff
@@ -113,17 +113,17 @@ class _SearchSuggestionPage extends State<SearchSuggestionPage> {
 
   //todo: get possible Rides from Algorithm
   Future<void> loadRides() async {
-    List<dynamic> data = await SupabaseManager.supabaseClient.from('drives').select('''
+    List<Map<String, dynamic>> data = await SupabaseManager.supabaseClient.from('drives').select('''
```

Comment:
the Exception is here aswell

Unhandled Exception: type 'List' is not a subtype of type 'List<Map<String, dynamic>>'